test/unit-test/FreeRTOS_IP_Utils/FreeRTOS_IP_Utils_utest.c

```diff
@@ -75,11 +74,110 @@ extern NetworkInterface_t xInterfaces[ 1 ];
 extern UBaseType_t uxLastMinBufferCount;
 extern size_t uxMinLastSize;
 
-BaseType_t xNetworkInterfaceInitialise_returnTrue( NetworkInterface_t * xInterface )
+/* ============================ Stubs Functions =========================== */
+
```

Comment:
We should move stub functions to stub file

Reply:
Updated.